lsst/ts/laser/csc.py

```diff
@@ -31,33 +32,45 @@ class LaserDetailedState(enum.IntEnum):
         Corresponds to the propagating state.
 
     """
-    DISABLEDSTATE = 1
-    ENABLEDSTATE = 2
-    FAULTSTATE = 3
-    OFFLINESTATE = 4
-    STANDBYSTATE = 5
-    PROPAGATINGSTATE = 6
+    DISABLEDSTATE = SALPY_TunableLaser.TunableLaser_shared_DetailedState_DisabledState
+    ENABLEDSTATE = SALPY_TunableLaser.TunableLaser_shared_DetailedState_EnabledState
+    FAULTSTATE = SALPY_TunableLaser.TunableLaser_shared_DetailedState_FaultState
+    OFFLINESTATE = SALPY_TunableLaser.TunableLaser_shared_DetailedState_OfflineState
+    STANDBYSTATE = SALPY_TunableLaser.TunableLaser_shared_DetailedState_StandbyState
+    PROPAGATINGSTATE = SALPY_TunableLaser.TunableLaser_shared_DetailedState_PropagatingState
 
 
-class LaserCSC(BaseCsc):
+class LaserErrorCode(enum.IntEnum):
```

Comment:
As per LTS-441 error codes for the in dome calibration system should be in the 7301-7700 range. Please, modify accordingly.